### ISSUE TITLE
Badge

### ISSUE THREAD

**Issue by Angelk90** [DamascenoRafael/reminders-menubar#24]
Hi @DamascenoRafael ,
It would be interesting to implement such a thing:

<img width="52" alt="Schermata 2021-08-06 alle 11 37 30" src="https://user-images.githubusercontent.com/20476002/128490857-b76705ff-6267-4019-a330-1676008cf55c.png">

**Comment by DamascenoRafael** [comment]
Hey @Angelk90, if you go into the settings gear and then under "Appearance" you can select an option to show the reminder count with due date for today. 

![image](https://user-images.githubusercontent.com/9842644/128649284-665328b5-46e6-4e5a-882c-4189bc79be53.png)

Were you imagining something different from this?

**Comment by Angelk90** [comment]
@DamascenoRafael : Yes I know, I advised instead of using that way to use a badge as seen in the image above.

P.s.
I did the following test, I started the program via xcode, I put a reminder with expiry today, but the number above does not appear.
How come you can't see it.

<img width="507" alt="Schermata 2021-08-09 alle 14 44 20" src="https://user-images.githubusercontent.com/20476002/128708049-c445174f-d299-46a2-ab48-4187102bbf54.png">

**Comment by DamascenoRafael** [comment]
@Angelk90 to add a due date to a reminder you need to do it through Apple's own Reminders app.
It is not yet possible to add an expiration date via Reminders Menu Bar.

When the reminder has an expiration date, that date is displayed under its title.

![image](https://user-images.githubusercontent.com/9842644/128796938-bb754e3e-8595-4c44-b50d-0a317733cec3.png)

Can you open the Apple Reminders app and add a reminder with an expiration date for today to see if it's working?

On Apple Reminders you should have something like:

![image](https://user-images.githubusercontent.com/9842644/128797117-ee6e54aa-56c5-4a20-b814-a2ec307e6884.png)

**Comment by Angelk90** [comment]
@DamascenoRafael : Ok tried, it works even if I would add reminder expiration option.

Returning to the original speech that to show the count of expiring reminders using a badge as in the figure.
![](https://user-images.githubusercontent.com/20476002/128490857-b76705ff-6267-4019-a330-1676008cf55c.png)

P.s.
What do you think about #23?

**Comment by DamascenoRafael** [comment]
The app now has more MenuBar customization options, but there are no plans to change the count to a badge.
